Remove outdated info from plugin starter kit (#8)
PostHog/posthog-plugin-starter-kit: README.md

- [ ] Optional: Add a `logo.png` file to give this plugin its own logo.
- [ ] Create a README.MD for your plugin which follows [our documentation guidelines](https://posthog.com/docs/plugins/build). 

> If you're looking for inspiration, then check out plugins source code from [data pipelines](https://posthog.com/docs/cdp).

Reviewer: Hmm, this link doesn't seem to go anywhere all that useful? I don't have a better idea off the top of my head though. Maaaaybe https://github.com/PostHog/?q=plugin&type=all&language=&sort= ?

Author: agreed, I was thinking they can check out plugins there, which have links to repos - which allows them to read what the plugin does first, pick if they are looking to do a destination or ingestion filtering or event transformation on the left, so in theory better than just linking to a mix of plugins
```suggestion
If you're looking for inspiration, then check out existing plugins and their source code from [data pipelines](https://posthog.com/docs/cdp).
```

When you're done, you can submit your plugin to our integration library so that it can be used by other users, including those on PostHog Cloud. 

To submit your plugin, use the support option in PostHog Cloud.

> We'll review the plugin to ensure it's secure, performant, and adheres to best practices and isn't already accomplisahable by other ways. Then, we add it to our official repository and make it available for everyone to use!

Reviewer: ```suggestion
We'll review the plugin to ensure it's secure, performant, adheres to best practices and isn't already doing something that can be accomplished by other means. Then we'll add it to our official repository and make it available for everyone to use!
```
